.travis.yml

```diff
@@ -0,0 +1,16 @@
+language: erlang
+otp_release:
+  - R16B03-1
+before_script:
+  - sudo apt-get update -qq
+  - sudo apt-get install postgresql-9.3-postgis-2.1 postgresql-contrib
+addons:
+  postgresql: "9.3"
+script:
+  - sudo chmod 777 /var/run/postgresql/
+  - ./setup_test_db.sh
+  - nohup ./start_test_db.sh &
```

Comment:
yes, `start_test_db.sh` blocks, but I think just `&` may be enough. Anyway, this is not a principal question.